docs/sources/tempo/configuration/grafana-alloy/tail-sampling/_index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Sampling
+menuTItle: Sampling
+description: Use sampling to optimize sampling decisions
+weight:
+aliases:
+- /docs/tempo/grafana-alloy/tail-based-sampling
+- ../tail-based-sampling/
+---
+
+# Sampling
+
+Grafana Tempo provides an inexpensive solution that aims to ingest and store the traces that provide maximum observability across your application estate.
```

Comment:
not sure about the word "inexpensive" and it's connotations - perhaps just leave out?

Reply:
Maybe cost effective instead of inexpensive? 
```suggestion
Grafana Tempo provides a cost-effective solution that aims to ingest and store the traces that provide maximum observability across your application estate.
```

Reply:
that sounds good to me:)